google/install-chrome/README.md

```diff
@@ -77,5 +77,121 @@ The following output values are available:
 - `${{ tasks.chrome.values.chrome-directory }}`
 - `${{ tasks.chrome.values.chromedriver-binary }}`
 - `${{ tasks.chrome.values.chromedriver-directory }}`
+- `${{ tasks.chrome.values.start-virtual-display-binary }} `
+- `${{ tasks.chrome.values.is-virtual-display-running-binary }} `
 
-The `chromedriver.*` values are only available when chromedriver is installed.
+The `chromedriver-*` values are only available when chromedriver is installed. The `*-virtual-display-*` values are only available when `headless` is false.
+
+# Using headless Chrome
+
+By default, `google/install-chrome` is configured with `headless: true`. You'll need to make sure you've configured your tool of choice to run use headless Chrome. Here's an example with Selenium and Ruby:
```

Comment:
```suggestion
By default, `google/install-chrome` is configured with `headless: true`. You'll need to make sure you've configured your tool of choice to run using headless Chrome. Here's an example with Selenium and Ruby:
```

Reply:
Ope thanks, I had re-worded this. Initially it was very testing-specific but I think people use browsers for other things than tests (web scraping, etc). I must've skipped over this word in my head